test/controllers/remote_execution_controller_test.rb

```diff
@@ -64,17 +62,16 @@ def test_customized_errata_install_with_errata_id_shows_new
       bulk_errata_ids =
         {
           included: {
-            ids: []
+            ids: [katello_errata(:security).errata_id]
           }
```

Comment:
Was it broken before cause this change seems right. 😕

Reply:
Yeah, I was a bit surprised that was blank too :shrug:  It did complain at me about it while I was fixing the tests, so I had to add that.